ci-operator/step-registry/cucushift/installer/check/control-plane-machinesets/cucushift-installer-check-control-plane-machinesets-commands.sh

```diff
@@ -51,6 +51,34 @@ function hasCPMS() {
     return ${ret}
 }
 
+# check if the cluster is by IPI or UPI
+# return 0 if it is an IPI cluster, otherwise 1
+function isIPI() {
+    #oc get machines -n openshift-machine-api -o json | jq -r '.items[].metadata.labels."machine.openshift.io/cluster-api-machine-role"' | grep master
+    # TODO: alternatively, do "oc get cm -n openshift-config openshift-install", 
+    # after UPI workflows removed referencing the step "ipi-install-times-collection"
+    oc get cm -n openshift-config openshift-install
```

Comment:
I suggest to append `-o yaml`, so that next time, if we hit some unexpected result, we can directly get the cm's data.